pkgs/servers/home-assistant/custom-components/miraie/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  miraie-ac,
+  aiomqtt,
+  nix-update-script,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "rkzofficial";
+  domain = "miraie";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "rkzofficial";
+    repo = "ha-miraie-ac";
+    tag = "v${version}";
+    hash = "sha256-UAhgnRTmXydEuo27VVbqlAi118b4vynuIljZR4x44lE=";
+  };
+
+  dependencies = [
+    miraie-ac
+    aiomqtt
+  ];
+
+  pythonRemoveDeps = [ "asyncio" ];
+
+
```

Comment:
```suggestion
  ignoreVersionRequirement = [ "asyncio" ];
```

Reply:
I'm still getting the same error with this option.

Reply:
Then you need to drop it from manifest.json. E.g. using a jq filter.

Reply:
@mweinelt I've pulled the new upstream release that fixes this.